content/kb/troubleshooting-unable-to-mount-an-nfs-backup-target.md

```diff
@@ -0,0 +1,94 @@
+---
+title: "Troubleshooting: Unable to access an NFS backup target"
+author: James Lu
+draft: false
+date: 2022-05-27
+categories:
+  - "backup"
+---
+
+## Applicable versions
+
+All Longhorn versions.
+
+## Prerequisite
+
+Longhorn supports only NFS versions 4.0, 4.1, and 4.2. Users should ensure that both their backup server and client support one of the supported versions.
+
+### How to determine which NFS version is used
+
+* Server side:
+
+Check which versions of NFS are currently enabled.
+
+```bash
+  # cat /proc/fs/nfsd/versions
+```
+
+* Client side:
+
+What versions the NFS mount is configured to support:
+
+```bash
+  # nfsstat -m
+```
+
+## Symptom 1
```

Comment:
We can describe the symptom and then provide possible root causes and solutions.

For example,
- Symptom
  The error message on backup UI page complained
  ```
  error exit status 32: vers=4.2: Failed to execute: mount [-t nfs4 -o nfsvers=4.2 -o actimeo=1 192.168.121.170:/opt/nfs-server /var/lib/longhorn-backupstore-mounts/192_168_121_170/opt/nfs-server], output mount.nfs4: mounting 192.168.121.170:/opt/nfs-server failed, reason given by server: No such file or directory
  , error exit status 32: Cannot mount using NFSv4"
  ```

- Troubleshooting
  - Possible Reason 1
    Check if the configuration in `/etc/exports` contains `fsid=0`....
    If YES, explain why and how to resolve it.
  - Possible Reason 2
     ....

Reply:
@derekbit I rewrote Symptom 1 and does it look like what you mentioned?